metricbeat/tests/system/test_mysql.py

```diff
@@ -0,0 +1,45 @@
+import os
+import metricbeat
+from nose.plugins.attrib import attr
+
+MYSQL_FIELDS = metricbeat.COMMON_FIELDS + ["mysql"]
+
+MYSQL_STATUS_FIELDS = ["clients", "cluster", "cpu", "keyspace", "memory",
+                     "persistence", "replication", "server", "stats"]
+
+
+class RedisInfoTest(metricbeat.BaseTest):
```

Comment:
Usually I prefer calling these just Test so they are easy to reference in the CLI and we don't forget to update them when copying code around :-).